main.tex

```diff
@@ -35,6 +35,71 @@ \section{Ausgangsbedingungen}
+Die Projekte Facebook Zero (Bereitstellung einer kostenlosen Internetverbindung solange man auf Facebooks Webseiten ist) und ``Facebook für jedes Handy" (eine Version von Facebook für feature Phones) wurden 2010 und 2011 eingeführt.
+Die verwendete Technologie will Facebook nun allgemein anwendbar machen, damit andere Dienstleistungen diese auch benutzen können.
+
+\medskip
+
+Mit dem letzen Punkt sagt Facebook, dass Sie glauben das die Ausbreitung des Internets die Aufgabe der Industrie ist, denen Facebook mit Konzepten wie Internet.org zu Seite stehen möchte.
+
+Dieses neue Konzept wurde in wie fern umgesetzt: Menschen in Entwicklungsländern sollen zuerst nur den kostenlosen Zugang zu grundlegenden Dienstleistungen wie Facebook haben.
+Für diese Herangehensweise wurden zwei Begründungen gegeben:
+\medskip
+Nicht jegliche Dienstleistung kann kostenlos bereitgestellt werden, da die dahinterstehende Anbieter Profit erzielen müssen. 
+Aber durch das kostenlose Bereitstellen grundlegender Anwendungen könnte man gewährleisten, dass die meisten Menschen Zugang zum Internet haben und die Industrie trotzdem gleichzeitig den größten wirtschaftlichen Profit erzielt (wodurch diese wiederum mehr Zugang zum Internet bieten kann).
+Obwohl Zuckerberg sagt, dass er keine spezifische Gruppe von grundlegenden Internet Dienstleistungen vorgibt, beschreibt er welche Anwendungen er als grundlegend ansieht.
+Seiner Meinung nach sind es die Dienstleistungen, die nicht viel Daten verbrauchen (hauptsächlich Text basierende und einfach gehaltene Apps) und durch welche anderer Inhalt entdeckt werden kann.
+Dienstleistungen die unter diesen Kriterien fallen sind zum Beispiel Messengers, soziale Netzwerke, Suchmaschinen und Wikipedia.(\cite[5]{HumanRight})
+Also ist Facebook nach Zuckerbergs Meinung eine grundlegende Internet Dienstleistung, die vorrangig zu anderen Anwendungen kostenlos zur Verfügung gestellt werden sollte.
+
+Aber durch das kostenlose Bereitstellen grundlegender Anwendungen könnte man gewährleisten, dass die meisten Menschen Zugang zum Internet haben und die Industrie trotzdem gleichzeitig den größten wirtschaftlichen Profit erzielt (wodurch diese wiederum mehr Menschen den Zugang zum Internet ermögliche können).
+Obwohl Zuckerberg sagt, dass er keine spezifische Gruppe von grundlegenden Internet Dienstleistungen vorgibt, beschreibt er welche Anwendungen er als grundlegend ansieht.
+
+\begin{quote}
+``many people who have never experienced the internet don’t know what a data plan is or why they’d want one. 
+However, most people have heard of services like Facebook and messaging and they want access to them. If we can provide people 
+with access to these services, then they’ll discover other content they want and begin to use 
+and understand the broader internet." - \textcite[4]{HumanRight}
+\end{quote}
+
+Facebooks Plan ist es diese Punkte unter dem Projekt Internet.org Industrie übergreifend zu bearbeiten.
```

Comment:
Delet this.

Reply:
was genau?

Reply:
I finde das gut, rundet das Zitat ab

Reply:
@SuFischer  99-108 are dublicate 
At least in part